sql/moz-fx-data-shared-prod/telemetry/retention_clients/view.sql

```diff
@@ -0,0 +1,127 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.telemetry.retention_clients`
+AS
+WITH clients_last_seen AS (
+  SELECT
+    cls.submission_date,
+    cls.client_id,
+    cls.sample_id,
+    cls.normalized_channel,
+    mozfun.norm.os(cls.os) AS normalized_os,
+    COALESCE(
+      mozfun.norm.windows_version_info(cls.os, cls.os_version, cls.windows_build_number),
+      NULLIF(SPLIT(cls.normalized_os_version, ".")[SAFE_OFFSET(0)], "")
+    ) AS normalized_os_version,
+    cls.country,
+    cls.is_dau,
+    cls.is_wau,
+    cls.is_mau,
+    mozfun.bits28.retention(cls.days_seen_bits, cls.submission_date) AS retention_seen,
+    mozfun.bits28.retention(
+      cls.days_active_bits & cls.days_seen_bits,
+      cls.submission_date
+    ) AS retention_active,
+    cls.days_seen_bits,
+    cls.days_active_bits,
+  FROM
+    `moz-fx-data-shared-prod.telemetry.clients_last_seen_v2` cls
+  WHERE
+    cls.submission_date = @submission_date
```

Comment:
Can you use `@submission_date` on a VIEW declaration?

Reply:
I'm not sure, I will check.

Reply:
Cannot use `@submission_date` on a VIEW declaration. Will make a table instead